docs/tutorials/where_do_I_get_papers.md

```diff
@@ -0,0 +1,125 @@
+# Where to get papers
+
+## OpenReview
+
+You can use papers from [https://openreview.net/](https://openreview.net/) as your database!
+Here's a helper that fetches a list of all papers from a selected conference (like ICLR, ICML, NeurIPS), queries this list to find relevant papers using LLM, and downloads those relevant papers to a local directory which can be used with paper-qa on the next step. Install `openreview-py` with
+
+```bash
+pip install paper-qa[openreview]
+```
+
+and get your username and password from the website. You can put them into `.env` file under `OPENREVIEW_USERNAME` and `OPENREVIEW_PASSWORD` variables, or pass them in the code directly.
+
+```python
+from paperqa import Settings
+
+# these settings require gemini api key you can get from https://aistudio.google.com/
+# import os; os.environ["GEMINI_API_KEY"] = os.getenv("GEMINI_API_KEY")
+# 1Mil context window helps to suggest papers. These settings are not required, but useful for an initial setup.
+settings = Settings.from_name("openreview")
+
+question = "What is the progress on brain activity research?"
+
+
+from paperqa.contrib.openreview_paper_helper import OpenReviewPaperHelper
```

Comment:
Can you move this import to the top of the snippet?

Or did you have it down here for a reason